packages/gatsby-theme-docs/src/components/side-by-side.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import { dimensions } from '@commercetools-docs/ui-kit/src/design-system';
+import { usePageData } from '../hooks/use-page-data';
+
+// allows the first and second child of this container to float side by side in large desktop sizes
+const SideBySideContainer = styled.div`
+  margin-top: 0 !important; /* a stupid "important-war" with the UI kit's stack components */
```

Comment:
Using important is bad practice (IMHO) but to be able to use UI kit stacks I had to use !important to override the !important in the UI Kit.  Which is a bad idea, too concerning what a UI kit is good for.  

A Stack that's more optimized for responsive body content would be helpful instead of having to do hacks like this.  

If you have a good idea how to solve this layout situation more elegantly I'm very happy, as you see in the comment it's not making me happy.  I wanted to use UI kit stuff because you asked for it in all previous code but it's not an ideal fit here. 

**Problem Statement**: The Side By Side content is sometimes an entry in the UI Kit stack (e.g. in API docs).  the UI kit stack forces the top spacing and uses !important. 
But the SideBySide is responsive, i.e. the second item can sometimes be to the right of the first and sometimes under it.  So we need different spacing "logic" here - the UI Kit stacks and inlines are not intended for responsive content (without this override the right item would not be top aligned with the left item).

Reply:
Thanks for the explanation. I added the reasoning as a comment so that it's clear what's it for.